airbyte-workers/named_pipes/np_source/Dockerfile

```diff
@@ -1,4 +1,4 @@
 FROM alpine:latest
 COPY run.sh /tmp/run.sh
-ENV AIRBYTE_ENTRYPOINT="/tmp/run.sh"
-ENTRYPOINT /tmp/run.sh
+ENV AIRBYTE_ENTRYPOINT=">&2 /tmp/run.sh"
```

Comment:
This is here to demonstrate the StdErr redirection works. Will remove before merging.